code/modules/mob/living/simple_animal/simple_animal.dm

```diff
@@ -414,7 +414,7 @@
 	else if(meat_type && (stat == DEAD))	//if the animal has a meat, and if it is dead.
 		if(istype(O, /obj/item/weapon/kitchen/knife))
 			harvest()
-	else
+	else if(istype(O) && istype(user) && !O.attack(src, user))

```

Comment:
This is so that you can give special attack effects for attacking simple animals - before, it would always default to standard attack code, since this fully overrides attackby and does not do a parent call.